composer.json

```diff
@@ -22,14 +22,17 @@
     "require-dev": {
         "pear/archive_tar": "^1.4.3",
         "phpunit/phpunit": "^5.7.23 || ^6.4.3",
+        "psr/http-message": "^1.0",
         "zendframework/zend-coding-standard": "~1.0.0",
         "zendframework/zend-crypt": "^3.2.1",
+        "zendframework/zend-diactoros": "^1.8",
         "zendframework/zend-servicemanager": "^2.7.8 || ^3.3",
         "zendframework/zend-uri": "^2.6"
     },
     "suggest": {
         "zendframework/zend-crypt": "Zend\\Crypt component, for encryption filters",
         "zendframework/zend-i18n": "Zend\\I18n component for filters depending on i18n functionality",
+        "zendframework/zend-diactoros": "Zend\\Diactoros for filtering files uploaded through PSR-7 middleware",
```

Comment:
Why was this added here? I can not find any dependency to zend-diactoros? (outside the tests)

Reply:
https://github.com/zendframework/zend-filter/pull/70/files#diff-7218516065f9379b209dac8a0b351324R197

https://github.com/zendframework/zend-filter/pull/70/files#diff-7218516065f9379b209dac8a0b351324R179

Calling functions to get names and to perform actual move.
As in other comment, reasoning was that if PSR payload is given, then there must be already implementation available. If not, suggest a library that will allow usage of it.

Reply:
> Calling functions to get names and to perform actual move.

The interface `Psr\Http\Message\UploadedFileInterface` is needed, not zend-diactoros.

Reply:
Done.